packages/react-components/react-dialog/src/components/DialogBody/renderDialogBody.tsx

```diff
@@ -1,12 +1,18 @@
-import * as React from 'react';
-import { getSlots } from '@fluentui/react-utilities';
+/* eslint-disable jsdoc/check-tag-names */
+/** @jsxRuntime classic */
+/** @jsx createElement */
+/* eslint-enable jsdoc/check-tag-names */
```

Comment:
Can't we update SWC config instead?

https://swc.rs/docs/configuration/compilation#jsctransformreact

Reply:
Yeaaah, I've tested that already. only way to adopt swc for builds is if we migrate everything together.